test/arbitrum/JPEGzMainnetOracle.t.sol

```diff
@@ -0,0 +1,505 @@
+    assertEq(owner, user);
+    assertEq(address(treasury).balance, fee);
+  }
+
+  function testMint_ShouldGiveBackETH_WhenFeeIsHigh() public {
+    uint256 fee = ethVault.getMintFee(1 ether);
+    vm.startPrank(user);
+    vm.deal(user, 100 ether);
+    ethVault.createVault();
+    ethVault.addCollateralETH{value: 10 ether}();
+    ethVault.mint{value: 10 ether}(1 ether);
+
+    (uint256 id, uint256 collateral, uint256 debt, address owner) = ethVault
+      .vaults(1);
+    assertEq(id, 1);
+    assertEq(collateral, 10 ether);
+    assertEq(debt, 1 ether);
+    assertEq(owner, user);
+    assertEq(address(treasury).balance, fee);
+    assertTrue(user.balance > 10 ether - ethVault.getMintFee(1 ether));
+  }
+
+  function testMint_ShouldFail_WhenFeeIsNotPaid() public {
+    vm.startPrank(user);
+    vm.deal(user, 100 ether);
+    ethVault.createVault();
+    ethVault.addCollateralETH{value: 10 ether}();
+    vm.expectRevert("VaultHandler::mint: mint fee less than required");
+    ethVault.mint(1 ether);
+
+    (uint256 id, uint256 collateral, uint256 debt, address owner) = ethVault
+      .vaults(1);
+    assertEq(id, 1);
+    assertEq(collateral, 10 ether);
+    assertEq(debt, 0 ether);
+    assertEq(owner, user);
+  }
+
+  function testMint_ShouldMint_WhenEnoughCAP() public {
+    // setup
+    orchestrator.executeTransaction(
+      address(ethVault),
+      0,
+      "setMinimumTCAP(uint256)",
+      abi.encode(20 ether)
+    );
+    uint256 fee = ethVault.getMintFee(20 ether);
+    vm.startPrank(user);
+    vm.deal(user, 100 ether);
+    ethVault.createVault();
+    ethVault.addCollateralETH{value: 10 ether}();
+
+    // execution
+    vm.expectRevert("VaultHandler::mint: mint amount less than required");
+    ethVault.mint{value: fee}(1 ether);
+
+    ethVault.mint{value: fee}(20 ether);
+    ethVault.mint{value: fee}(1 ether);
+
+    //asserts
+    (, , uint256 debt, ) = ethVault.vaults(1);
+    assertEq(debt, 21 ether);
+
+    // setup
+    vm.stopPrank();
+    orchestrator.executeTransaction(
+      address(ethVault),
+      0,
+      "setMinimumTCAP(uint256)",
+      abi.encode(30 ether)
+    );
+
+    // execution
+    vm.startPrank(user);
+    vm.expectRevert("VaultHandler::mint: mint amount less than required");
+    ethVault.mint{value: fee}(1 ether);
+    ethVault.mint{value: fee}(9 ether);
+
+    // asserts
+    (, , debt, ) = ethVault.vaults(1);
+    assertEq(debt, 30 ether);
+  }
+
+  function testBurnTCAP_ShouldBurn_WhenFeeIsPaid(uint96 _tcapAmount) public {
+    // checks
+    if (_tcapAmount < 1 ether) {
+      return;
+    }
+
+    // setUp
+    uint256 t = 1 wei;
+    orchestrator.executeTransaction(
+      address(ethVault),
+      0,
+      "setMinimumTCAP(uint256)",
+      abi.encode(1 ether)
+    );
+    vm.startPrank(user);
+    uint256 requiredCollateral = ethVault.requiredCollateral(_tcapAmount);
+    uint256 mfee = ethVault.getBurnFee(_tcapAmount);
+    uint256 bfee = ethVault.getBurnFee(_tcapAmount);
```

Comment:
Can you test the value of bfee like the way I've requested you to test the mintFee?